### PR TITLE
PD-197980: revise backstage system information

### PR DESCRIPTION
**Pull request by SergiiKryvenko**
## Jira Issue #

[PD-197980](https://bazaarvoice.atlassian.net/browse/PD-197980)

## What Are We Doing Here?

Add/revise information about components on [the Bаckstage system](https://backstage.prod.bazaarvoice.com/catalog/default/component/).

## How to Test and Verify

1. Check out this PR
2. Check if the info on [the Bаckstage system](https://backstage.prod.bazaarvoice.com/catalog/default/component/) displays correctly: [priam](https://backstage.prod.bazaarvoice.com/catalog/default/component/priam)

### Level

`Low`

### Required Testing

`Smoke`

## Code Review Checklist

- [ ] Testing instructions are included. If not, make sure you leave us a line or two for the reason.

- [ ] Well documented, including updates to any necessary markdown files. When
  we inevitably come back to this code it will only take hours to figure out, not
  days.

- [ ] Make sure PR has only the changes required for the Jira issues.

- [ ] Have someone from a different region than you review your changes. If not, make sure to leave a comment on why it is not required

- [ ] If it's applicable, document the deployment plan for the changes.

### CODE REVIEW

File: catalog-info.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    backstage.io/techdocs-ref: 'url:https://github.com/bazaarvoice/Priam/blob/3.0.x/README.md'
+  links:
+    - url: https://bazaarvoice.atlassian.net/wiki/spaces/DEV/pages/6793731414/Restore+a+Cassandra+cluster+from+backup+using+Priam
+      title: Restore a Cassandra cluster from backup using Priam
+      icon: docs
+  tags:
+    - emodb
+    - java
+    - priam
+    - dropwizard
```
Comment (mike-unitskyi) [inline catalog-info.yaml:20]: would add cassandra

**Review by mike-unitskyi — CHANGES_REQUESTED** [review]
looks good

File: catalog-info.yaml (hunk 1 above)
Comment (SergiiKryvenko) [reply catalog-info.yaml:20]: done

File: catalog-info.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  name: priam
+  description: 'Priam is a process/tool that runs alongside Apache Cassandra to automate the following:
+                Backup and recovery (Complete and incremental), 
+                Token management, 
+                Configuration, 
+                Support AWS environment'
+  annotations:
+    backstage.io/techdocs-ref: 'url:https://github.com/bazaarvoice/Priam/blob/3.0.x/README.md'
+  links:
+    - url: https://bazaarvoice.atlassian.net/wiki/spaces/DEV/pages/6793731414/Restore+a+Cassandra+cluster+from+backup+using+Priam
```
Comment (mike-unitskyi) [inline catalog-info.yaml]: would remove this or migrate whole file to the internal github repository

Comment (SergiiKryvenko) [reply catalog-info.yaml]: I changed it to the Netflix/Priam GitHub doc

**Review by mike-unitskyi — APPROVED** [review]

**Pull request merged**